### ISSUE TITLE
Segmentation fault when hover on the top slider

### ISSUE THREAD

**Issue by atomic** [gnumdk/lollypop#323]
Here's the *log* of the crash when executed from terminal:
```
** (lollypop:20551): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-LEK9S7LsFD: Connection refused
No module named 'pylast'
    - Scrobbler disabled
    - Auto cover download disabled
    - Artist informations disabled
$ sudo pip3 install pylast
No module named 'wikipedia'
Advanced artist informations disabled
$ sudo pip3 install wikipedia
Namespace WebKit2 not available
WebKit support disabled

** (lollypop:20551): CRITICAL **: Unable to determine array length for 0x7fe530061f30

** (lollypop:20551): CRITICAL **: Unable to determine array length for 0x7fe52c062c80

** (lollypop:20551): CRITICAL **: Unable to determine array length for 0x7fe5140474e0

** (lollypop:20551): CRITICAL **: Unable to determine array length for 0x7fe52005a0e0

** (lollypop:20551): CRITICAL **: Unable to determine array length for 0x7fe51004f4e0
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/lollypop/toolbar_title.py", line 116, in _on_progress_motion_notify
    slider_width = self._progress.style_get_property('slider-width') / 2
TypeError: style_get_property() takes exactly 3 arguments (2 given)
[1]    20551 segmentation fault (core dumped)  lollypop
```

**Comment by gnumdk** [comment]
What Gtk version?

**Comment by atomic** [comment]
I have libgtk2.24 and libgtk3.14 installed, elementaryos freya 0.3.1.

**Comment by gnumdk** [comment]
Fixed in coming 0.9.60

**Comment by KOT040188** [comment]
I upgraded lollypop to 0.9.300 and got this error. lollypop does not start. I'm upset.

**Comment by gnumdk** [comment]
>TypeError: style_get_property() takes exactly 3 arguments (2 given)
This one?

**Comment by gnumdk** [comment]
https://github.com/gnumdk/lollypop/issues/1255
